completion_aggregator/tracking.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Tracking and analytics events for completion aggregator activities.
+"""
+
+from eventtracking import tracker
+
+from . import compat
+
+
+TRACKER_BI_EVENT_NAME_FORMAT = u'edx.bi.user.{agg_type}.{event_type}'
+TRACKER_EVENT_NAME_FORMAT = u'edx.completion.aggregator.{event_type}'
+TRACKER_VALID_EVENT_TYPES = ('completed', 'started', )
```

Comment:
Set is better, although there might be no performance difference.

```suggestion
TRACKER_VALID_EVENT_TYPES = {'completed', 'started'}
```